app/components/profile/complexity-level/ComplexityLevelForm.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { Component } from 'react';
+import { intlShape } from 'react-intl';
+import type { $npm$ReactIntl$IntlFormat } from 'react-intl';
+import classes from './ComplexityLevelForm.scss';
+import classnames  from 'classnames';
+import BeginnerLevel from '../../../assets/images/complexity-level/beginner-level.inline.svg';
+import AdvancedLevel from '../../../assets/images/complexity-level/advanced-level.inline.svg';
+
+type Props = {|
+
+|}
+type State = {|
+
+|}
+
+const levels = [
+  {
+    name: 'simple',
+    image: <BeginnerLevel />,
+    description: 'Simplest experience possible. No previous knowledge in Blockchain required. Highly friendly to on-board beginners, and for users that prefer simplicity',
+  },
+  {
+    name: 'advanced',
+    image: <AdvancedLevel />,
+    description: 'I have a some understanding of blockchain and how cryptography is used to power both the blockchain itself and wallet software. I am okay with seeing options and functionality that critically depend on my understanding of these concepts.',
+  }
+];
+
+class ComplexityLevel extends Component<Props, State> {
+
+  static contextTypes: {|intl: $npm$ReactIntl$IntlFormat|} = {
+    intl: intlShape.isRequired,
+  };
+
+  render() {
+    return (
+      <>
+        <div className={classes.component}>
+          <div className={classes.header}>
+            <h1>Level of Interface Complexity</h1>
```

Comment:
(multiple places)

Prefer to create strings using defineMessages to make sure they get picked up by translations (recall: `npm run prod:build && npm run translations:manage` to update the translation file)